uaa/src/test/java/org/cloudfoundry/identity/uaa/mock/clients/ClientAdminEndpointDocs.java

```diff
@@ -83,7 +83,7 @@ class ClientAdminEndpointDocs extends AdminClientCreator {
         fieldWithPath("jwks").constrained("Optional only, if jwks_uri is used. Required otherwise.").type(STRING).description("A valid JSON string according JSON Web Key Set standard, see [RFC 7517](https://www.rfc-editor.org/rfc/rfc7517), e.g. content of /token_keys endpoint from UAA"),
         fieldWithPath("jwks_uri").constrained("Optional only, if jwks is used. Required otherwise.").type(STRING).description("A valid URI to token keys endpoint. Must be compliant to jwks_uri from [OpenID Discovery](https://openid.net/specs/openid-connect-discovery-1_0.html)."),
         fieldWithPath("kid").constrained("Optional only, if a single JWK should be deleted, else ignored.").type(STRING).description("If change mode is set to `"+ClientJwtChangeRequest.ChangeMode.DELETE+"`, the `id of the key` that will be deleted. The kid parameter is only possible if jwks configuration is used."),
```

Comment:
Minor suggestion:

If change mode is set to DELETE, it specifies `the id of the key` that will be deleted. The kid parameter is only applicable when jwks configuration is used.